### ISSUE TITLE
Infinite spinning wheel after video upload fail

### ISSUE THREAD

**Issue by iuriimoz** [onfido/onfido-ios-sdk#61]
### What was the expected behaviour?

Onfido SDK should hide spinning wheel or dismiss the flow if video upload fails.

### What happened instead?

Infinite spinning wheel is displayed and UI is blocked

### Version info:

- Cocoapods: Onfido 10.1.0
- iOS: 12.1
- Xcode: 10.1
- Device: iPhone 6s
- Device language: English

### Integration configuration:

```
let config = try! OnfidoConfig.builder()
    .withToken(token)
    .withApplicantId(applicantId)
    .withDocumentStep()
    .withFaceStep(ofVariant: .video)
    .build()

let onfidoFlow = OnfidoFlow(withConfiguration: config)
    .with(responseHandler: responseHandler)

do {
    let onfidoRun = try onfidoFlow.run()
    onfidoRun.modalPresentationStyle = .formSheet // to present modally
    self.present(onfidoRun, animated: true, completion: nil)
} catch let error {
    self.showErrorMessage(forError: error)
}
```

### Steps to reproduce:

1) Launch Onfido flow and scan any document
2) Disable internet connection
3) Record video and submit it
4) "Connection lost" alert appears, press "OK"
5) Infinite spinning wheel is displayed and UI is blocked

![1](https://user-images.githubusercontent.com/1536479/49895765-02a9e300-fe5a-11e8-891d-1fbe75cfc0d7.PNG)
![2](https://user-images.githubusercontent.com/1536479/49895766-03427980-fe5a-11e8-8812-f499561502fa.PNG)

**Comment by ramprasad-parajuli** [comment]
Hey @iuriimoz, Thanks for flagging this, we will try to replicate it with a similar device and we will get back to you ASAP

Regards, Ramprasad Parajuli

**Comment by anuragajwani** [comment]
Hi @iuriimoz,

We have reproduced the issue and have a fix being reviewed. The fix will be included in the next release.

Sorry for the inconvenience,
Anurag

**Comment by iuriimoz** [comment]
Great news. Thanks @anuragajwani

**Comment by anuragajwani** [comment]
@iuriimoz we have released the fix on 10.2.0 https://github.com/onfido/onfido-ios-sdk/releases/tag/10.2.0

**Comment by anuragajwani** [comment]
Closing issue for now. This should be fixed on Onfido iOS SDK 10.2.0 and forward.
